### PR TITLE
Fix plot bug

### PR DESCRIPTION
**Pull request by jaxonwillard**

### CODE REVIEW

File: Exam1/Problem1/main.py [hunk 1]
```diff
@@ -46,17 +45,14 @@ def GetMagnitude(x,y,x_2,y_2):
     y_help_list = []
     x_error_list =[]
     y_error_list = []
-    for dt in np.arange(0.0, max_time,t_setp):
+    for dt in np.arange(0.0, max_time,t_step):
         # for dt in range()
         
-        psi =  psi + Psi_dot *t_setp
```
Comment (jaxonwillard) [inline Exam1/Problem1/main.py:52]: This was the problem that I found. This (and the next two lines) are computing the next step, but using the current dt as the parameter for GetExactCoord, so the discrete function was always one step ahead.

Comment (jaxonwillard) [reply Exam1/Problem1/main.py:52]: So I moved it to the end of this for loop. If you want to see the difference, you can set t_list = [0.01] and max_time = 0.05. You will see that they start in the same place

**Comment by jaxonwillard** [comment]
This looks nice when max_time=10. But if you set max time to a high number like 100, for some reason the errors increase over time. Not sure why

**Review by danielkboyer — APPROVED** [review]
Makes sense, so my understanding is it was incrementing the new position and plotting that point but it should of been plotting the current point and incrementing the position after.

**Comment by danielkboyer** [comment]
> This looks nice when max_time=10. But if you set max time to a high number like 100, for some reason the errors increase over time. Not sure why

I think this is expected, because the longer it goes the more the errors start to add up. If there is a .1 error for the first circle, the next circle should be .2.

**Pull request merged**